atlas_densities/densities/measurement_to_density.py

```diff
@@ -69,8 +70,10 @@ def compute_region_densities(
         The index is the sorted list of all region identifiers.
     """
     densities = []
-    for set_ in hierarchy_info["descendant_id_set"]:
-        mask = np.isin(annotation, list(set_))
+    total = len(hierarchy_info["descendant_id_set"])
```

Comment:
generally, we don't have libraries `print()` or output to the terminal, since then it breaks them being able to be used from other code.  Since this change isn't needed for dealing w/ the inconsistent layer labels, could we remove it?

Reply:
Agree, and the tqdm bar is already showing the total.
Addressed in https://github.com/BlueBrain/atlas-densities/pull/20/commits/59f2a60f4bf7ede74d1226d54531b62d7fa1e7bb

Reply:
I would consider the tqdm bar to also be printing output to the terminal that the caller can't can't control, but it looks like that shipped has already sailed in other parts of this project.  Thanks for the change.